packages/@react-spectrum/table/src/Table.tsx

```diff
@@ -524,10 +525,11 @@ function TableRow({item, children, ...otherProps}) {
           styles,
           'spectrum-Table-row',
           {
+            'is-active': isPressed && allowsSelection,
```

Comment:
IMO it would be better to add a separate `usePress` within the TableRow component rather than returning `isPressed` through all of these hooks.